docs/core/logging.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Logging
+description: Core utility
+---
+
+Logger provides an opinionated logger with output structured as JSON.
+
+**Key features**
+
+* Capture key fields from Lambda context, cold start and structures logging output as JSON
+* Log Lambda event when instructed (disabled by default)
+    - Enable explicitly via annotation param
+* Append additional keys to structured log at any point in time
+
+## Initialization
+
+Powertools extends the functionality of Log4J. Below is an example log4j2.xml file, with the LambdaJsonLayout configured.
```

Comment:
I would highlight both the filename `log4j2.xml` and XML attribute `LambdaJsonLayout`, draws the focus of the reader easier if they are scanning the docs. In addition I would highlight the line 5 where the XML attribute is.